pkg/csi/service/vanilla/controller.go

```diff
@@ -1762,6 +1983,29 @@ func (c *controller) createFileVolume(ctx context.Context, req *csi.CreateVolume
 					"failed to create volume. Error: %+v", err)
 			}
 		}
+		attributes := make(map[string]string)
```

Comment:
Do we have to remove the code above this? Where are again filtering out datastores and calling `CreateFileVolumeUtil`? the volume task is already registered. So I think it is not needed to create another task.

Reply:
For the logic which is not guarded by the `TopologyAwareFileVolume` FSS, It's better to leave it unchanged. I think FVT tests are run with FSS set to false.